core/build.gradle

```diff
@@ -3,6 +3,8 @@ dependencies {
     compile project(':gateway')
     compile project(':store')
 
+    compile "com.github.Discord4J.Stores:stores-jdk:$jdkstores_version"
```

Comment:
I recommend the `compile "com.discord4j.stores:stores-jdk:master-SNAPSHOT"` coordinates instead to avoid lowercase/uppercase confusion and for consistency throughout our (WIP) docs.